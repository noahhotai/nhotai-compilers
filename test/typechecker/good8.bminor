
main: function integer ( a: integer, b: integer ) =
{
	
}

testfunc: function void ( z: integer ) =
{
	// Nested integer operators.
	y: integer = z*(z*10 + 55%6 + z/10);

}