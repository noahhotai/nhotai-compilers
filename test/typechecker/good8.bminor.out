Program parsed successfully.
z resolves to param 0
z resolves to param 0
z resolves to param 0
Program resolved successfully.
Program type checked successfully.
